packages/strapi-design-system/src/Combobox/ComboboxList.js

```diff
@@ -1,8 +1,12 @@
 import { useEffect } from 'react';
 import PropTypes from 'prop-types';
-
 import { maintainScrollVisibility } from './utils';
 
+/**
+ * @deprecated
+ * This component was deprecated in 1.6.3. It will be removed in the next major release 2.0.0.
+ */
```

Comment:
@joshuaellis not sure if this is all I need to do to deprecate the file 🤔. You mentioned you had a function to cherry pick, but not sure I understood exactly what that would be for.

Reply:
This will be done later, so thanks for leaving the note! 😄 

What do you think about removing the effect so anyone using the component has it fixed?